.changeset/smooth-mice-dance.md

```
@@ -0,0 +1,8 @@
+---
+"logixlysia": patch
+---
+
+- Optimized log formatting performance by replacing multiple `.replaceAll()` calls with a single-pass regex replacement in `formatLogOutput` and `formatTimestamp`.
+- Optimized `getIp` string slicing to reduce array allocations.
+- Fixed a bug where padded HTTP methods (e.g., `GET    `) lost their color formatting.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)